test/parallel_api/ranges/transform2_ranges_factory_sycl.pass.cpp

```diff
@@ -37,22 +38,20 @@ main()
     auto lambda1 = [](auto i) { return i * i; };
```

Comment:
Is it not enough to add
`using namespace oneapi::dpl::experimental::ranges;`
after `#if _ENABLE_RANGES_TESTING` inside main ?